components/eamxx/src/physics/ace/eamxx_ace_process_interface.hpp

```diff
@@ -0,0 +1,100 @@
+#ifndef EAMXX_ACE_PROCESS_INTERFACE_HPP
+#define EAMXX_ACE_PROCESS_INTERFACE_HPP
+
+#include <torch/script.h>
```

Comment:
It would be nice to use some pimpl idiom to hide torch implementation in the cpp file, and avoid propagating the torch includes.